Singer.API/ClientApp/src/app/modules/core/services/auth.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable, Inject } from '@angular/core';
 import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { JwtHelperService } from '@auth0/angular-jwt';
-import { Observable, Subject } from 'rxjs';
+import { Observable, Subject, BehaviorSubject } from 'rxjs';
```

Comment:
Als we Subject niet meer gebruiken waarom staat deze dan nog tussen de imports?

Reply:
Will be fixed.